src/nvim/vim.h

```diff
@@ -27,9 +27,6 @@ Error: configure did not run properly.Check auto/config.log.
 # endif
 #endif
 
-/* user ID of root is usually zero, but not for everybody */
-#define ROOT_UID 0
```

Comment:
> would (or should) know that root's UID will always be 0

This is the reason I made the change; I'm all for avoiding magic numbers, but the UID/GID of root seems too common to necessitate a macro.